src/file/FileUtils.js

```diff
@@ -41,6 +41,11 @@ define(function (require, exports, module) {
         StringUtils         = require("utils/StringUtils"),
         Dialogs;            // This will be loaded asynchronously
 
+    // Asynchronously loading Dialogs to avoid the circular dependency
+    require(["widgets/Dialogs"], function (dialogsModule) {
+        Dialogs = dialogsModule;
+    });
+
```

Comment:
I moved this to the top just for clarity.  This change isn't related to any of the rename stuff, and doesn't affect the timing of when this module is loaded.